clang/test/Analysis/NewDeleteLeaks-PR60896-shared.cpp

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=core,cplusplus,unix -verify %s
+// expected-no-diagnostics
+
```

Comment:
Can we combine this test into the unique_ptr one? These two sort of cover the same subjects, so it would make more sense to me to test them in a single file, nearby to each other.